jsapp/js/components/formSubScreens.js

```diff
@@ -10,7 +10,7 @@ import ProjectSettings from './modalForms/projectSettings';
 import FormMedia from './modalForms/formMedia';
 import {PROJECT_SETTINGS_CONTEXTS} from '../constants';
 import FormMap from './map';
-import RESTServices from './RESTServices';
+import RESTServices from './RESTServices/RESTServices';
```

Comment:
optional: as part of the upcoming manual linter rules, name the file `RESTServices/index.ts` to clearly signal the entrypoint (in that folder) and then it can be imported simply as
```suggestion
import RESTServices from './RESTServices';
```